src/txmempool.cpp

```diff
@@ -67,6 +68,16 @@ struct update_fee_delta
     int64_t feeDelta;
 };
 
+struct update_fee
+{
+    explicit update_fee(int64_t _fee) : fee(_fee) { }
```

Comment:
Wow, this is more C++ than I understand!

Reply:
It's just a lens, though a very rigid and uninteresting one. I just copied the mechanism used for updating feeDelta for fees.